internal/cli/kraft/lib/add/add.go

```diff
@@ -0,0 +1,126 @@
+package add
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"kraftkit.sh/cmdfactory"
+	"kraftkit.sh/pack"
+	"kraftkit.sh/packmanager"
+	"kraftkit.sh/unikraft"
+	"kraftkit.sh/unikraft/app"
+	"kraftkit.sh/unikraft/lib"
+)
+
+type AddOptions struct {
+	Workdir    string `long:"workdir" short:"w" usage:"workdir to add the package to"`
+	Kraftfile  string `long:"kraftfile" short:"K" usage:"Set an alternative path of the Kraftfile"`
+	FromServer bool   `long:"from-server" usage:"Force package manager to use manifests of the server"`
```

Comment:
To be consistent with the other subcommands which have a similar flag:
```diff
- FromServer bool   `long:"from-server" usage:"Force package manager to use manifests of the server"`
+ NoUpdate     bool   `long:"no-update" usage:"Do not update package index before running the build"`
```
Also ensure that entries are sorted alphabetically.